geth/params/cluster.go

```diff
@@ -0,0 +1,55 @@
+package params
+
+type subClusterData struct {
+	Number      int      `json:"number"`
+	Hash        string   `json:"hash"`
+	StaticNodes []string `json:"staticnodes"`
+}
+
+type clusterData struct {
```

Comment:
We may also think about flattening this structure:
```go
type cluster struct {
  NetworkID int `json:"networkID"`
  Prod bool `json:"production"` // or custom type based on int; we may have dev, prod, test
  StaticNodes []string `json:"staticnodes"`
}
```

Reply:
going to add bootnodes to this structure in the upcoming prs, and i will include simplification in the same pr